Tests failing on live

```
FAILED project/npda/tests/test_csv_upload.py::test_dates_with_short_year - assert False
FAILED project/npda/tests/test_csv_upload.py::test_thyroid_and_coeliac_dates_within_90_days_before_diagnosis_pass_validation - TypeError: Invalid value '2022-07-10' for dtype 'datetime64[us]'
FAILED project/npda/tests/test_csv_upload.py::test_thyroid_and_coeliac_dates_earlier_than_90_days_before_diagnosis_fail_validation - TypeError: Invalid value '2022-07-10' for dtype 'datetime64[us]'
```

I was only able to reproduce this locally by rebuilding the container so I suspect some transitive library upgrades?

`pandas` is not bound to a particular version! v3 was released yesterday so I suspect that's causing the breakage

--- a/requirements/requirements.txt
+++ b/requirements/requirements.txt
@@ -16,8 +16,8 @@ django-citext==1.0.0
 django-auto-logout==0.5.1
 docutils==0.20.1
 geopandas==1.0.1
-markdown
-pandas
+markdown==3.10.1
+pandas==2.3.3
 openpyxl==3.1.5
 psycopg2-binary==2.9.9
 whitenoise==6.6.0
@@ -32,7 +32,7 @@ phonenumbers==8.13.31
 plotly==5.19.0
 
 # NHS number
-nhs-number
+nhs-number==1.3.6
 
 # profiling
 django-silk==5.4.3

